src/modules/vtol_att_control/vtol_att_control_params.c

```diff
@@ -213,12 +213,24 @@ PARAM_DEFINE_FLOAT(VT_F_TRANS_DUR, 3.0f);
  *
  * @unit s
  * @min 0.00
- * @max 10.00
+ * @max 20.00
  * @increment 1
  * @decimal 2
  * @group VTOL Attitude Control
  */
-PARAM_DEFINE_FLOAT(VT_B_TRANS_DUR, 2.0f);
+PARAM_DEFINE_FLOAT(VT_B_TRANS_DUR, 4.0f);
+
+/**
+ * Back transition MC motor ramp up time
+ *
+ * @unit s
+ * @min 0.00
+ * @max 20.00
+ * @increment 1
+ * @decimal 2
+ * @group VTOL Attitude Control
+ */
+PARAM_DEFINE_FLOAT(VT_B_TRANS_RAMP, 2.0f);
```

Comment:
Isn't this linked to the forward and back transition period? The motor ramp up time should be hard-linked to that, not driven by an unrelated parameter that people easily forget.